File: cfgs/java_cfgs_foo/SMALLEST_SUM_CONTIGUOUS_SUBARRAY.java
Nodes
1;special;;;
2;statement;int min_ending_here = 2147483647;14;
3;statement;int min_so_far = 2147483647;15;
4;statement;int i = 0;16;
5;conditional;foo_1 < foo_2;17;
6;conditional;foo_1 > 0;19;
7;statement;min_ending_here = foo_1[foo_2];19;
8;statement;min_ending_here += foo_1[foo_2];20;
9;statement;min_so_far = fun_1(foo_1, foo_2);21;
10;statement;i++;18;
11;statement;return foo_1;23;
12;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;11;
5;7;8;
6;9;
7;9;
8;10;
9;5;
10;12;
11;

File: cfgs/pythoncfgs/SMALLEST_SUM_CONTIGUOUS_SUBARRAY.py
Nodes
1;statement;foo_1 = foo_2.maxsize;10;
2;statement;foo_1 = foo_2.maxsize;11;
3;conditional;fun_1(foo_1);12;
4;statement;return foo_1;18;
5;conditional;foo_1 > 0;13;
6;statement;foo_1 += foo_2[foo_3];16;
7;statement;foo_1 = fun_1(foo_2, foo_3);17;
8;statement;foo_1 = foo_2[foo_3];14;
9;exit;;
Edges
0;2;
1;3;
2;4;5;
3;9;
4;6;8;
5;7;
6;3;
7;7;
8;